resources/envoy.yaml

```diff
@@ -30,10 +30,42 @@ static_resources:
             address: adapter
             port_value: 18000
       http2_protocol_options: {}
+      transport_socket:
+        name: "envoy.transport_sockets.tls"
+        typed_config: 
+          "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
+          common_tls_context:
+            tls_params:
+              tls_minimum_protocol_version: TLSv1_2
+              tls_maximum_protocol_version: TLSv1_2
+            tls_certificates:
+              private_key:  
+                filename: "/home/wso2/security/mg.key"
```

Comment:
Check if this can be derived from env variables

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/1582/commits/391606c1e87d9aad9da15f06caffd3a309a468ac